backend/postgres/db/migrations/000001_initial.up.sql

```diff
@@ -0,0 +1,70 @@
+CREATE TABLE IF NOT EXISTS instances (
```

Comment:
Any reason `NOT EXISTS` is used here? In cases like migration (especially creation) being critical path, this may skip creating a table when one already exists with different schema.  Reentrancy here has too much of an edge case.